Example/FinancialConnections Example/FinancialConnections Example/Playground/PlaygroundView.swift

```diff
@@ -176,6 +176,9 @@ struct PlaygroundView: View {
         }
         .navigationTitle("Playground")
         .navigationBarTitleDisplayMode(.inline)
+        .onAppear {
+            UIScrollView.appearance().keyboardDismissMode = .interactive
```

Comment:
Does this only affect `PlaygroundView`? Or can it also affect the Auth Flow? _(if it can affect the auth flow that would be subpar because we always want to test the actual experience the users would have integrating in any app)_

Reply:
ChatGPT 😅 :

> Yes, setting UIScrollView.appearance().keyboardDismissMode = .interactive will affect all instances of UIScrollView within your app. The appearance() proxy is a way to configure default values for UI elements across the entire application, so any scroll view will adopt this keyboard dismiss mode unless it's specifically overridden in an individual scroll view instance.

---

@mats-stripe  If we are to blindly follow LLM's, what would be the explanation of how this works? How would calling `UIScrollView.appearance()` magically reference the correct thing? Not saying its impossible, but even the API feels very odd if its globally setting something, yet only affecting one thing?

There's also `.scrollDismissesKeyboard(.interactively)`?

Reply:
This also doesn't increase my confidence that its not global?

https://developer.apple.com/documentation/uikit/uiappearance

<img width="921" alt="Screenshot 2024-07-04 at 5 18 18 PM" src="https://github.com/stripe/stripe-ios/assets/105514761/1e62ad60-4c51-493c-a107-d32084a46b5a">

Reply:
I tested this out, and as you guessed, changing the `UIScrollView.appearance()` here affects the app behaviour globally 😅 

I just pushed a change to dismiss the keyboard when a drag gesture is recognized, which works just as well and is local to only the playground view!